src/osmf/lib/HDSPlayer.as

```diff
@@ -208,10 +282,14 @@ package
      */
     private function onPlayComplete(event:TimeEvent):void
     {
+      if (_captionLabel.visible)
+      {
+        _captionLabel.visible = false;
+      }
```

Comment:
Visibility of captions text box is set again on replay. Hence it is removed from here.